source/vocab/enums.ttl

```diff
@@ -473,3 +475,58 @@
 :SoundContent a owl:Class;
     rdfs:label "Ljudinnehåll"@sv, "Sound content"@en;
     owl:equivalentClass bf2:SoundContent .
+
+## For serials {{{
+
+:languagePeriod a owl:ObjectProperty ;
+    #rdfs:domain :Serial ;
+    :category :pending ;
+    rdfs:label "language period"@en, "språkperiod"@sv;
+    rdfs:range :LanguagePeriod .
+
+:LanguagePeriod a owl:Class ;
+    rdfs:subClassOf :TemporalQualification ;
+    rdfs:label "language period"@en, "språkperiod"@sv .
+
+:politicalTendencyPeriod a owl:ObjectProperty ;
+    #rdfs:domain :Serial ;
+    :category :pending ;
+    rdfs:label "political tendency period"@en, "period av politisk tendens"@sv;
+    rdfs:range :PoliticalTendencyPeriod .
+
+:PoliticalTendencyPeriod a owl:Class ;
+    rdfs:subClassOf :TemporalQualification ;
+    rdfs:label "political tendency period"@en, "period av politisk tendens"@sv .
+
+:politicalTendency a owl:ObjectProperty ;
+    skos:related :intendedAudience ;
+    rdfs:label "political tendency"@en, "politisk tendens"@sv;
+    :category :pending ;
+    sdo:domainIncludes :PoliticalTendencyPeriod ;
+    rdfs:range :PoliticalTendency .
+
+:PoliticalTendency a owl:Class ;
+    rdfs:subClassOf :Concept ;
+    rdfs:label "political tendency"@en, "politisk tendens"@sv .
+
+:frequencyPeriod a owl:ObjectProperty ;
+    rdfs:label "frequency period"@en, "frekvensperiod"@sv;
+    rdfs:subPropertyOf :pubFrequency .
+
+:FrequencyPeriod a owl:Class ;
+    rdfs:label "Frequency Period"@en, "Frekvensperiod"@sv;
+    rdfs:subClassOf :PubFrequency, :TemporalQualification .
+
+:firstIssueDate a owl:DatatypeProperty ;
+    rdfs:subPropertyOf :firstIssue, :startDate ;
+    rdfs:label "first issue date"@en, "första numrets datum"@sv;
+    sdo:domainIncludes :FormerTitle, :TemporalEntity ;
```

Comment:
`:TemporalEntity` seems like something that is too abstract too have a `firstIssueDate`/`lastIssueDate`?
Should be `startDate`/`endDate`?

Reply:
Yes, this is an anomaly. It was either `SerialEntity` or generalize the date properties (and deviate from e.g. `PubFrequency`). But I'll make a consolidation attempt!

Reply:
Note though that is is not `TemporalEntity` that "has" these issue-dates; it is these issue-date-properties that are declared as describing a `TemporalEntity`. Which is still "over-reaching", but true. (It says nothing about "every temporal entity".)

Reply:
The point of these two is to specify (as a subproperty of start-/end-date), for the "temporal aspect" (here a newspaper serial), that the dates are given for issues (c.f. [`bf:firstIssue`](https://id.loc.gov/ontologies/bibframe.html#p_firstIssue) and `bf:lastIssue`). They can be put on multiple kinds of temporal entities, so i "land-grabbed" rather than cherry-picked them.

We can refine these later on, as `sdo:domainIncludes` is just a "usage hint". We should certainly come to a shared understanding and agreement upon the whole temporality notion of course. I added some more notes on that in ef2cff4b19e49943ece453a49c2fdb7348f8161e.